inspirehep/modules/workflows/tasks/submission.py

```diff
@@ -373,11 +374,20 @@ def prepare_files(obj, eng):
     if not obj.files:
         return
 
+    def _define_filename(paper, file_name):
+        """Define the filename in according to the article source"""
```

Comment:
👍 
bad code < bad code commented < 'good code' < 'good code' commented < good code that needs no comments